doc/src/manual/profile.md

```diff
@@ -336,6 +336,23 @@ and how much garbage it collects each time. This can be enabled with
 [`GC.enable_logging(true)`](@ref), which causes Julia to log to stderr every time
 a garbage collection happens.
 
+### Allocation Profiler
+
+The allocation profiler records the stack trace, type, and size of each
+ allocation while it is running. It can be invoked with
+ [`Profile.Allocs.@profile`](@ref).
```

Comment:
```suggestion
The allocation profiler records the stack trace, type, and size of each
allocation while it is running. It can be invoked with
[`Profile.Allocs.@profile`](@ref).
```